crates/sui-framework/sources/crypto/hash.move

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::hash {
```

Comment:
Here, I think it's good to have one module/hash function as we do with signature schemes for consistency.

The Move stdlib breaks this rule by having just `hash`, but we can choose to fix it in our vendored stdlib if we like. Alternatively (and perhaps better), we can choose to exclude the stdlib `hash` module altogether and expose our own modules for the hash functions in the Move stdlib that we want.